corehq/apps/hqwebapp/views.py

```diff
@@ -705,13 +705,135 @@ def jserror(request):
+                and current_project_description != new_project_description):
+            domain_object.project_description = new_project_description
+            domain_object.save()
+
+        message += ((
+            "software plan: {software_plan}\n"
+        ).format(
+            software_plan=Subscription.get_subscribed_plan_by_domain(domain),
+        ))
+
+        debug_context.update({
+            'self_started': domain_object.internal.self_started,
+            'has_handoff_info': bool(domain_object.internal.partner_contact),
+            'project_description': domain_object.project_description,
+        })
+
+    subject = '{subject} ({domain})'.format(subject=report['subject'], domain=domain)
+
+    if full_name and not any([c in full_name for c in '<>"']):
+        reply_to = '"{full_name}" <{email}>'.format(**report)
+    else:
+        reply_to = report['email']
+
+    # if the person looks like a commcare user, fogbugz can't reply
+    # to their email, so just use the default
+    if settings.HQ_ACCOUNT_ROOT in reply_to:
+        reply_to = settings.SERVER_EMAIL
+
+    message += "Message:\n\n{message}\n".format(message=report['message'])
+    if post_params.get('five-hundred-report'):
+        extra_message = ("This message was reported from a 500 error page! "
+                         "Please fix this ASAP (as if you wouldn't anyway)...")
+        extra_debug_info = (
+            "datetime: {datetime}\n"
+            "Is self start: {self_started}\n"
+            "Has Support Hand-off Info: {has_handoff_info}\n"
+            "Project description: {project_description}\n"
+            "Sentry Error: {sentry_error}\n"
+        ).format(**debug_context)
+        traceback_info = cache.cache.get(report['500traceback']) or 'No traceback info available'
+        cache.cache.delete(report['500traceback'])
+        message = "\n\n".join([message, extra_debug_info, extra_message, traceback_info])
+
+    email = EmailMessage(
+        subject=subject,
+        body=message,
+        to=[to_email],
+        headers={'Reply-To': reply_to},
+        cc=other_recipients
+    )
+
+    if uploaded_file:
+        filename = uploaded_file.name
+        content = uploaded_file.read()
+        email.attach(filename=filename, content=content)
+
+    # only fake the from email if it's an @dimagi.com account
+    is_icds_env = settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS
+    if is_dimagi_email(report['username']) and not is_icds_env:
+        email.from_email = report['username']
+    else:
+        email.from_email = to_email
+
+    return email
+
+
 @method_decorator([login_required], name='dispatch')
 class BugReportView(View):
     def post(self, req, *args, **kwargs):
         email = self._get_email_message(
             post_params=req.POST,
             couch_user=req.couch_user,
-            uploaded_file=req.FILES.get('report_issue')
+            uploaded_file=req.FILES.get('feature_request')
```

Comment:
this should support both right? report_issue and feature_request?

Reply:
The `BugReportView` should only support `report_issue` since that is from the modal it uses. Likewise, `feature_request` should only be in `SolutionsFeatureRequestView`